nais/dev-gcp.yml

```diff
@@ -124,6 +135,10 @@ spec:
       value: "service.name=k9-punsj,service.namespace=k9saksbehandling,deployment.environment=q,nais.backend=elastic-apm;grafana-lgtm"
 
     # Integrasjoner (Rest)
+    - name: SIF_ABAC_PDP_BASE_URL
+      value: https://sif-abac-pdp.dev-fss-pub.nais.io/sif/sif-abac-pdp/api/tilgangskontroll/k9
```

Comment:
Kjører ikke sif-abac-pdp i dev-gcp også?
Er det da ikke bedre å kalle service discovery url-en. 
Husk da outbound rules :)

Reply:
Jeg tenkte å begynne med at alt som gjelder k9 går mot fss-instansene for å dele cache der. Men det blir sikkert mindre latency om vi går mot gcp-instansene

Reply:
Jeg skjønner. 
Da må du huske å legge til sif-abac-pdp.dev-fss-pub.nais.io i external-host for å åpne for utgående trafikk.